addons/test_event_full/tests/test_event_mail.py

```diff
@@ -327,3 +327,17 @@ def test_event_mail_on_sale_confirmation(self):
                 "email_from": self.test_event.organizer_id.email_formatted,
             },
         )
+
+    def test_template_translation(self):
+        self.env['res.lang']._activate_lang('fr_BE')
+        test_event = self.env['event.event'].create({
```

Comment:
You may use ``self.test_event`` which normally comes with templates and schedulers defined in ``TestEventFullCommon``